src/cheri/insns/gchi_32bit.adoc

```diff
@@ -20,6 +20,7 @@ include::wavedrom/gchi.adoc[]
 
 Description::
 Copy the metadata (bits [CLEN-1:MXLEN]) of capability `cs1` into `rd`.
+It is not required that the input capability `cs1`  has its tag set to 1.
```

Comment:
I think these are implicit unless the listing says the tag must be set, but clarifying it seems fine to me.

Reply:
I agree that it is implicit. All other instructions of this class have this statement and only this one was missing it. So I added it for consistency reasons.